services/f-droid/f-droid.tester.js

```diff
@@ -0,0 +1,26 @@
+
+t.create('Package is found')
+  .get('/f-droid/version/org.pacien.tincapp.json')
+  .intercept(nock =>
+    nock('https://f-droid.org')
+      .get('/en/packages/org.pacien.tincapp/')
```

Comment:
The tests are red although and I do not know why.

```
$ npm run test:services -- --grep F-Droid

> gh-badges@1.3.0 test:services /home/user/shields
> HANDLE_INTERNAL_ERRORS=false mocha --delay lib/service-test-runner/cli.js "--grep" "F-Droid"

Running all service tests.


No secret data found at /home/user/shields/private/secret.json (see lib/server-secrets.js)
raven@2.4.2 alert: no DSN provided, error reporting disabled
0823224940 FsTokenPersistence configured with /home/user/shields/private/github-user-tokens.json
0823224940 Server is starting up: http://lib/service-test-runner/cli.js:80/
No fallback font set.
Unable to load fallback font. Using Helvetica-Bold instead.
  F-Droid
    Package is found
GET /f-droid/version/org.pacien.tincapp.json has FAILED with the following response:
      1) 
	[ GET /f-droid/version/org.pacien.tincapp.json ]
    Package is not found
GET /f-droid/version/org.pacien.tincapp.json has FAILED with the following response:
      2) 
	[ GET /f-droid/version/org.pacien.tincapp.json ]


  0 passing (827ms)
  2 failing

  1) F-Droid
       Package is found
         
	[ GET /f-droid/version/org.pacien.tincapp.json ]:

      AssertionError: expected { Object (name, value) } to deeply equal { Object (name, value) }
      + expected - actual

       {
      -  "name": "404"
      -  "value": "badge not found"
      +  "name": "F-Droid"
      +  "value": "v1.8asdsadsa"
       }
      
      at Object.pathMatch.matchJSON (node_modules/icedfrisby/lib/pathMatch.js:138:38)
      at _expect (node_modules/icedfrisby/lib/icedfrisby.js:583:10)
      at IcedFrisbyNock._invokeExpects (node_modules/icedfrisby/lib/icedfrisby.js:1261:26)
      at start (node_modules/icedfrisby/lib/icedfrisby.js:1244:12)
      at Request.runCallback [as _callback] (node_modules/icedfrisby/lib/icedfrisby.js:1131:16)
      at Request.self.callback (node_modules/request/request.js:185:22)
      at Request.<anonymous> (node_modules/request/request.js:1161:10)
      at IncomingMessage.<anonymous> (node_modules/request/request.js:1083:12)
      at endReadableNT (_stream_readable.js:1056:12)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickDomainCallback (internal/process/next_tick.js:218:9)

  2) F-Droid
       Package is not found
         
	[ GET /f-droid/version/org.pacien.tincapp.json ]:

      AssertionError: expected { Object (name, value) } to deeply equal { Object (name, value) }
      + expected - actual

       {
      -  "name": "404"
      -  "value": "badge not found"
      +  "name": "F-Droid"
      +  "value": "app not found"
       }
      
      at Object.pathMatch.matchJSON (node_modules/icedfrisby/lib/pathMatch.js:138:38)
      at _expect (node_modules/icedfrisby/lib/icedfrisby.js:583:10)
      at IcedFrisbyNock._invokeExpects (node_modules/icedfrisby/lib/icedfrisby.js:1261:26)
      at start (node_modules/icedfrisby/lib/icedfrisby.js:1244:12)
      at Request.runCallback [as _callback] (node_modules/icedfrisby/lib/icedfrisby.js:1131:16)
      at Request.self.callback (node_modules/request/request.js:185:22)
      at Request.<anonymous> (node_modules/request/request.js:1161:10)
      at IncomingMessage.<anonymous> (node_modules/request/request.js:1083:12)
      at endReadableNT (_stream_readable.js:1056:12)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickDomainCallback (internal/process/next_tick.js:218:9)

```
See https://f-droid.org/en/packages/org.pacien.tincapp/
and https://github.com/badges/shields/pull/1965/files#diff-63441b8c1c2f41111e85c4e116ac3804R10

Reply:
Resolved: The path is `/version` and not `/f-droid/version`